src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -786,19 +786,6 @@ public void parseWarnsAboutUnmatchedContentInEntry() throws IOException {
     }
 
     @Test
-    @Ignore("Ignoring because this is an edge case")
```

Comment:
It just doesn't work right now as @lenhard says. So there is no sense to test it at the moment.